app/scss/pages/_home.scss

```diff
@@ -0,0 +1,123 @@
+.rooms {
+	&__inner {
+		padding: 60px 0 95px;
+		@media (max-width : 680px) {
+			padding: 45px 0 65px;
+		}
+		@media (max-width : 320px) {
+			padding: 35px 0 45px;
+		}
+	}
+	&__title {
+		margin-bottom: 90px;
+		font-size: 25px;
+		font-weight: 400;
+		text-align: center;
+		@media (max-width : 680px) {
+			margin-bottom: 25px;
+		}
+		@media (max-width : 480px) {
+			font-size: 17px;
+		}
+	}
+}
+.room {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 70px;
+	position: relative;
+	&::before {
+		content: "";
+		border: 1px solid #14274A;
+		position: absolute;
+		height: 230px;
+		left: -35px;
+		bottom: 103px;
+		@media (max-width : 1170px) {
+			display: none;
+		}
+	}
+	@media (max-width : 990px) {
+		flex-wrap: wrap;
+		justify-content: center;
+	}
+	@media (max-width : 480px) {
+		margin-bottom: 35px;
+	}
+	&__info {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		margin-right: 70px;
+		@media (max-width : 990px) {
+			text-align: center;
+			margin-right: 0;
+		}
+		&-title {
+			font-family: AGaramondPro, sans-serif;
+			font-size: 50px;
+			letter-spacing: 0.02em;
+			font-weight: 700;
+			letter-spacing: 0.02em;
+			line-height: 1.1;
+			@media (max-width : 1160px) {
+				font-size: 40px;
+			}
+			@media (max-width : 1160px) {
+				font-size: 35px;
+			}
+			@media (max-width : 480px) {
+				line-height: 1;
+			}
+		}
+		&-text {
+			@media (max-width : 480px) {
+				font-size: 12px;
+			}
+		}
+		&-btn {
+			width: 170px;
+			height: 50px;
+			text-align: center;
+			@media (max-width : 990px) {
+				margin: 20px auto;
+			}
+		}
+	}
+	&__img {
+		img {
+			width: 650px;
+			height: 450px;
```

Comment:
incorrect
You have a case when you have two different images with differtnt width and height but you should make them equal witdh and height
Look at this article - https://medium.com/@kimcodeashian/simple-css-solutions-how-to-resize-images-with-different-dimensions-without-cropping-eb744e93246d

<img width="371" alt="Screenshot 2021-01-24 at 22 30 40" src="https://user-images.githubusercontent.com/13525860/105642698-e3cd0600-5e93-11eb-8f28-8e56239733a2.png">